projects/smoke_drv120/src/main.c

```diff
@@ -0,0 +1,57 @@
+#include <stdbool.h>
+
+#include "delay.h"
+#include "drv120_relay.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+// Toggle the following to select which data is logged
+#define ENABLE_PIN \
+  { .port = GPIO_PORT_A, .pin = 8 }
+#define STATUS_PIN \
+  { .port = GPIO_PORT_A, .pin = 6 }
+#define TIME_INTERVAL 500
+
+static void prv_error_handler(void *context) {
+  LOG_DEBUG("Error %s\n", "error occured");
```

Comment:
`Error error occured` isn't a very descriptive error message. With the DRV120 driver, this error handling callback will be called if the DRV120 signals undervoltage lockout or thermal shutdown, so I think that this would be a good message: `Warning: DRV120 signalled undervoltage lockout or thermal shutdown`. Also, it would be nice to use `LOG_WARN` instead of `LOG_DEBUG` here since it's a warning.